pkg/kudoctl/cmd/params/parse.go

```diff
@@ -0,0 +1,112 @@
+package params
```

Comment:
I prefer file names that are nouns, and we already have multiple `parser.go` files in the project. Let's stay consistent.

Reply:
Done.